src/templates/ApiReference.js

```diff
@@ -273,10 +273,14 @@ ReferenceSection.propTypes = {
 
 const AbsoluteEl = styled.div`
   position: absolute;
-  overflow-y: scroll;
+  overflow: hidden;
   height: calc(100% - 48px);
   width: 100%;
   bottom: 3.25rem;
+
+  &:hover {
+    overflow-y: scroll;
+  }
```

Comment:
Oh this is clever, it hides scrollbar on e.g. Windows?

Reply:
Yes!

Reply:
Ugly scrollbar though